perfkitbenchmarker/providers/__init__.py

```diff
@@ -31,69 +31,67 @@
 CLOUDSTACK = 'CloudStack'
 RACKSPACE = 'Rackspace'
 MESOS = 'Mesos'
+SOFTLAYER = 'SoftLayer'
 
 VALID_CLOUDS = (GCP, AZURE, AWS, DIGITALOCEAN, KUBERNETES, OPENSTACK,
-                RACKSPACE, CLOUDSTACK, ALICLOUD, MESOS)
+                SOFTLAYER, RACKSPACE, CLOUDSTACK, ALICLOUD, MESOS)
 
 
 _imported_providers = set()
 
 
 def LoadProviderFlags(providers):
-  """Imports just the flags module for each provider.
-
-  This allows PKB to load flag definitions from each provider to include in the
-  help text without actually loading any other provider-specific modules.
-
-  Args:
-    providers: series of strings. Each element is a value from VALID_CLOUDS
-        indicating a cloud provider for which to import the flags module.
-  """
-  for provider_name in providers:
-    normalized_name = provider_name.lower()
-    flags_module_name = '.'.join((__name__, normalized_name, 'flags'))
-    importlib.import_module(flags_module_name)
+    """Imports just the flags module for each provider.
```

Comment:
Sorry to be nit-picky, but we usually have a blank line between the one-line summary and the longer description, and between the longer description and the args.  

Also, we typically use two spaces for the indent. 

We generally follow the [google python style guide](https://google.github.io/styleguide/pyguide.html)